### DISCUSSION TITLE
Pre-training Qwen for better domain specific knowledge?

### DISCUSSION THREAD

**Discussion by Tejaswgupta** [QwenLM/Qwen#1211] [Q&A]
We saw lawyer-llama built by using SFT on a dataset of legal QA pairs. Recently Saul-LM which was fine-tuned on Mistral achieves good performance as well. 
However, I've seen Qwen-72B performing exceptionally well just with prompt tuning, would it be feasible to do continual pre-training and would it possibly improve accuracy and conversation quality?

Use cases:
- Adding clauses to draft
- Drafting using prompt
- Legal QA , for ex: legal limit of drinking or how to file for bankrupty
- Tool usage (to use tools like a retriever)
- Precedent(Case law) retrieval
